b9/src/ExecutionContext.cpp

```diff
@@ -189,7 +189,8 @@ StackElement ExecutionContext::interpret(const std::size_t functionIndex) {
     instructionPointer++;
     programCounter_++;
   }
-  throw std::runtime_error("Reached end of function");
+  //throw std::runtime_error("Reached end of function");
+  return StackElement(0);
```

Comment:
I think we should preserve the old behaviour: crash if we try to execute an end-section. Eventually, I suspect we'll be eliminating end-section entirely.

What we should do instead is, output an implicit
```lisp
(int-push-const 0)
(return)
```
at the end of the script's body.

Reply:
Once we start emitting push-0/return in the <body>, then it will be impossible for the <body> function to be empty. I bet this will fix our crash when JIT compiling the <body>.